### ISSUE TITLE
Quake mode available by default when computer starts

### ISSUE THREAD

**Issue by tolache** [microsoft/terminal#11177]
Currently, Quake mode is available using the `Win`+`` ` `` only after I run Windows Terminal at least once since the computer starts. It would be very handy in Windows Terminal Quake mode shortcut worked right after computer starts without having to first run Windows Terminal in normal mode.

**Comment by zadjii-msft** [comment]
Thanks for the suggestion! Please make sure to [search for duplicates](https://github.com/microsoft/terminal/issues?q=is%3Aissue+quake+start) when filing new issues. This is already being tracked by another issue on our repo - please refer to #9996 for more discussion.

/dup #9996
